src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/convenience/key_vault_credential.py

```diff
@@ -0,0 +1,26 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+#pylint: skip-file
+
+import adal
+import urllib
+
+from msrest.authentication import authentication
+
+from . import httpbearerchallenge
+from . import httpbearerchallengecache as challengecache
+
+class keyvaultcredential(authentication):
+
+    def __init__(self, token):
+        print('init')
+        self._token = token
+
+    def signed_session(self):
+        session = super(keyvaultcredential, self).signed_session()
+        session.headers['authorization'] = 'bearer {}'.format(self._token or '')
+        print('signed session w/ auth: {}'.format(session.headers['authorization']))
```

Comment:
Shouldn't print to stdout here.
Do you need to print the Authorization header? If so, consider using logging.